src/zcl_abapgit_environment.clas.abap

```diff
@@ -0,0 +1,36 @@
+CLASS zcl_abapgit_environment DEFINITION
+  PUBLIC
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    CLASS-METHODS is_cloud
```

Comment:
I would prefer to use the method name is_sap_cloud_plattform instead of is_cloud. Reason is that there are several types of clouds in SAP terminology. In case of we need to differenciate between them in a later point of time I would introduce the mentioned method name.

Reply:
okay, feel free to open PR renaming the method

Reply:
I will do the changes and open a PR to rename the method name